README.md

```diff
@@ -1 +1,67 @@
-# rhdh-plugin-export-overlays
+## What is the rhdh-plugin-export-overlays reposiotry?
+
+This repository contains workflows to package Backstage plugins (under the following npm scopes: @backstage-community, @red-hat-developer-hub and @roadiehq) as OCI images to be used as dynamic plugins within Red Hat Developer Hub (RHDH).
+
```

Comment:
Maybe some additional explanation about the fact that the content of the data is partitioned:
- per workspace, in distinct folders corresponding to disctinct monorepos hosted in some third-party GH repositories)
- per RHDH target release, based on the use of release branches: long-running branches named `release-x.y`, x.y being the RHDH release (1.5, 1.6, etc ).